src/views/explore/PhenotypeExplorer.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <strong>Compare these phenotypes ...</strong>
+
+  <!-- set A -->
+  <AppSelectTags
+    name="First set of phenotypes"
+    :options="getPhenotypes"
+    v-model="aPhenotypes"
+    placeholder="Select phenotypes"
+    :tooltip="multiTooltip"
+    @valueFunc="valueFunc"
+  />
+
+  <!-- set B -->
+  <AppFlex gap="small">
+    <strong>... to</strong>
+    <AppSelectSingle
+      name="Second set mode"
+      :options="bModeOptions"
+      v-model="bMode"
+    />
+    <AppSelectSingle
+      v-if="bMode.includes('genes')"
+      name="Second set taxon"
+      :options="bTaxonOptions"
+      v-model="bTaxon"
+    />
+  </AppFlex>
+
+  <AppSelectTags
+    v-if="bMode.includes('these phenotypes')"
+    name="Second set of phenotypes"
+    :options="getPhenotypes"
+    v-model="bPhenotypes"
+    placeholder="Select phenotypes"
+    :tooltip="multiTooltip"
+    @valueFunc="valueFunc"
+  />
+
+  <!-- example button -->
+  <AppButton text="try an example" design="small" @click="doExample()" />
+
+  <!-- run analysis -->
+  <AppButton text="Analyze" icon="bars-progress" @click="runAnalysis" />
+
+  <hr />
+
+  <!-- analysis status -->
+  <AppStatus v-if="status" :status="status" />
+
+  <!-- analysis results -->
+  <AppFlex v-else-if="results.matches.length">
+    <strong>Top 10 matches</strong>
+    <div
+      v-for="(match, index) in results.matches.slice(0, 10)"
+      :key="index"
+      class="match"
+    >
+      <AppRing
+        :score="match.score"
+        :min="results.minScore"
+        :max="results.maxScore"
+        v-tippy="'Similarity score'"
+      />
+      <div class="match-details">
+        <AppIcon
+          :icon="`category-${match.category}`"
+          v-tippy="startCase(match.category)"
+        />
+        <AppLink :to="match.id">
```

Comment:
So, when I run the example query I get a URL for this link that looks like: `/FBcv:0000366%20+%20FBcv:0000439%20+%20FBcv:0002041%20+%20FBcv:0002039`. 2.0 doesn't appear to accept sets of phenotypes like this, and I don't know if they exist in the database at all. Perhaps it'd be better to split on the "+" and link to each one individually?

Reply:
Here I am simply displaying the id the backend returns for the `id` from the `compare` endpoint:

<img width="483" alt="image" src="https://user-images.githubusercontent.com/8326331/159045056-59aef9c4-0492-4149-aeb3-fc7e1192a4fe.png">

For some reason it returns them as a `+` separated list. I kind of assumed this was just a result of me choosing and providing bad example phenotypes, and that it would work properly when using good phenotypes.

Nevertheless, it seems like the backend shouldn't return a single concatenated string like that for an `id` field which is purportedly supposed to be a single id.

Reply:
Yeah, this is still something I'm unclear about from a science standpoint...like, in variant databases, there's a distinction to be made between comparing multiple variants versus variant combinations. Variant combos are usually indicated with pluses like that, e.g. "BRAF V600 + BRAF K601E" would be the combination of two BRAF variants, that could have a very different effect from either of them individually.

Perhaps it does make sense for an ID to be a compound ID (i.e., multiple IDs combined with pluses) if that combination in itself is an entity, i.e. it has distinct behavior different from any specific ID. It'd be good to get some clarity from someone who's familiar with these phenotype representations and comparisons.

Reply:
Ah I see. I'll look into what 2.0 does for this case and file an issue on 3.0 to implement this if necessary.

Reply:
#89